plugins/modules/proxmox_kvm.py

```diff
@@ -1306,6 +1320,7 @@ def main():
                               sshkeys=module.params['sshkeys'],
                               startdate=module.params['startdate'],
                               startup=module.params['startup'],
+                              storage=module.params['storage'],
```

Comment:
This change seems unrelated. Is this a bugfix? If yes, it should be a separate PR.

Reply:
indeed it's not strictly mandatory for this change, but we probably need to allow user to define the storage target when creating  or  restoring a VM. I will remove this on the current PR & create a separate one!